### ISSUE TITLE
Save ( upload to server ) doesn't work after a while

### ISSUE THREAD

**Issue by jarvit** [humy2833/FTP-Simple#142]
Hello,

I've configured ftp-simple as a project and for a few minutes when i press a custom combination for upload to the server everything works fine.

After a while, if it will not upload anything for 4 or 5 minutes then when i try to save and upload it again it doesn't work and then i have to close and re-open the Visual Code.

Does anyone else have the same issue ?

Here is my Config:

[
	{
		"name": "NAME",
		"host": "hostname",
		"port": 21,
		"type": "ftp",
		"username": "user",
		"password": "password",
		"path": "/",
		"project": {
			"E:/WORK/project": {
				"path": "/",
				"save": true
			}
		},
		"ignore": [
			"/**/node_modules",
			"/**/*.class"
		],
		"confirm": false,
		"autosave": true
	}
]

**Comment by diakonos** [comment]
Yes, the same happens to me. To fix it, I execute the ftp-simple command: `Close all open FTP connections.`

**Comment by jarvit** [comment]
This is a good Trick, but the solution could be to run this command after 3/4 mins of idle. @humy2833

**Comment by humy2833** [comment]
Please update to version 0.6.4.
Does it happen the same after updating?

**Comment by jarvit** [comment]
Yes my version is 0.6.4 and the problem still occurs
